js/10-base.js

```diff
@@ -93,6 +93,17 @@ Liberapay.init = function() {
         ).prop('selected', true);
         history.pushState(null, null, location.pathname + href + location.hash);
     });
+    
+    $('input[data-required-if-checked]').each(function(index, value) {
+        var radio = $(value).attr('data-required-if-checked');
+        $($(radio).parents('form')).on('click', function(e) {
```

Comment:
Clicking isn't the only way to interact with a form. The proper solution is probably to listen for the [`change`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event) or [`input`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) event on the radio input(s).